### ISSUE TITLE
My server crashed with "TypeError: Cannot read property 'encode' of undefined"

### ISSUE THREAD

**Issue by jankeromnes** [espadrine/email-login#2]
Here is the full error message:

```
project/node_modules/email-login/src/registry.js:258
    fs.writeFile(accf, this.account[email].encode(), cb);
                                          ^

TypeError: Cannot read property 'encode' of undefined
    at Object.Registry.saveAccount (project/node_modules/email-login/src/registry.js:258:43)
    at project/node_modules/email-login/src/registry.js:198:18
    at FSReqWrap.oncomplete (fs.js:82:15)
```

**Comment by espadrine** [comment]
There was a change that fixed this issue in a4449140. I added a few patches to make the code more robust against this class of errors. Please upgrade to v0.1.2, and tell me if you find more issues.

**Comment by jankeromnes** [comment]
Thanks! I updated my version of email-login.

I'll close this for now, and re-open if I get the same error again.
